pkg/operator/certrotation/target.go

```diff
@@ -83,8 +85,8 @@ func (c TargetRotation) ensureTargetCertKeyPair(signingCertKeyPair *crypto.CA, c
 	return nil
 }
 
-func needNewTargetCertKeyPair(annotations map[string]string, caBundleCerts []*x509.Certificate, validity time.Duration, renewalPercentage float32) bool {
-	if needNewCertKeyPairForTime(annotations, validity, renewalPercentage) {
+func needNewTargetCertKeyPair(annotations map[string]string, signer *crypto.CA, caBundleCerts []*x509.Certificate, validity time.Duration, renewalPercentage float32) bool {
```

Comment:
in my own words, we rotate if
- our old CA is gone from the bundle (then we are pretty late to the renewal party)
- or the cert expired (then we are also pretty late)
- or we are over the renewal percentage of the validity, but only if the new CA at least 10% into its age. 

Maybe worth a go doc.

So in general we need to see a signing CA at least aged 10% within 1-percentage of the cert validity.

Hence, if the CAs are rotated too fast (like CA percentage around 10% or smaller), we will not hit the time to make use of the CA. Or if the cert renewal percentage is at 90%, there is not much time either.

So with a cert percentage of 75% and equally long CA and cert validities at the worst case we start at 85% of the cert to renew, trying again every minute.